packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -134,20 +148,29 @@ export class StripeAutomaticTaxConverter {
         invoicePreview
       );
 
-      // TODO FXA-6581: Write report to CSV
-      console.log('report:', report);
+      await this.writeReport(report);
+
+      console.log(subscriptionId);
```

Comment:
Should this console log remain?

Also, not relevant to this review, but was just curious if it was considered for the script to track which customers/subscriptions were skipped and for what reason?

Reply:
I added it so that there's _some_ output when the script is running otherwise it just pends quietly with no visible indication of progress.

Reply:
OK great, sounds good thank you.